dm/templates/dns_managed_zone/dns_managed_zone.py

```diff
@@ -22,7 +22,7 @@ def generate_config(context):
     managed_zone_name = context.properties.get('zoneName')
     dnsname = context.properties['dnsName']
     managed_zone_description = context.properties['description']
-    name_servers = '$(ref.' + context.properties['zoneName'] + '.nameServers)'
+    name_servers = '$(ref.' + context.env['name'] + '.nameServers)'
```

Comment:
What this is required for?

Reply:
resource name was 
```
 managed_zone = {	    managed_zone = {
        'name': context.env['name'],
```

But it was referenced as 
```
context.properties['zoneName'] 
```
which it not always the same